ui/app/templates/components/cluster-info.hbs

```diff
@@ -38,13 +38,13 @@
           {{/if}}
         {{else}}
           {{#if (has-permission 'status' routeParams='replication')}}
-            <nav class="menu">
+           <nav class="menu">
```

Comment:
same here, it looks like you removed the indendentation but then added two indentations below the `nav` element.  I would expect one indendentation after the if and then one after the nav.